src/components/App/App.scss

```diff
@@ -23,6 +23,11 @@
 body {
   background: url(../../../public/images/main-background.jpg) var(--primary-dark-color);
   background-size: cover;
```

Comment:
Since you mentioned thinking about what you could add visually, I love the background - almost wish it was zoomed in and moving REAL slowly?

Reply:
🤯